tests/unit/engine/test_setup_transform.py

```diff
@@ -95,102 +90,132 @@ def _setup(self, _stage: str | None = None) -> None:
+        dataset = DummyDataset(transform=transform)
+        dataloader = DataLoader(dataset, batch_size=1)
+        model = DummyModel()
+        # before the setup_transform is called, the dataset should have the custom transform
+        assert dataset.transform == transform
+        Engine._setup_transform(model, dataloaders=dataloader)  # noqa: SLF001
+        # after the setup_transform is called, the model should have the custom transform
+        assert model.transform == transform
+
     # test if the user-specified transform is used when passed to the datamodule
     def test_user_specified_transform(self) -> None:
         """Tests if the user-specified transform is used when passed to the datamodule."""
-        custom_transform = Resize((100, 100))
-        engine = Engine()
+        custom_transform = Transform()
         datamodule = DummyDataModule(transform=custom_transform)
         model = DummyModel()
         # assert that the datamodule uses the custom transform before and after setup_transform is called
         assert datamodule.train_transform == custom_transform
         assert datamodule.eval_transform == custom_transform
-        engine._setup_transform(model, datamodule=datamodule)  # noqa: SLF001
+        Engine._setup_transform(model, datamodule=datamodule)  # noqa: SLF001
         assert datamodule.train_transform == custom_transform
         assert datamodule.eval_transform == custom_transform
+        assert model.transform == custom_transform
 
     # test if the user-specified transform is used when passed to the datamodule
     def test_user_specified_train_transform(self) -> None:
         """Tests if the user-specified transform is used when passed to the datamodule as train_transform."""
         model = DummyModel()
-        default_size = model.configure_transforms().size
-        custom_transform = Resize((100, 100))
-        engine = Engine()
+        custom_transform = Transform()
         datamodule = DummyDataModule(train_transform=custom_transform)
         # before calling setup, train_transform should be the custom transform and eval_transform should be None
         assert datamodule.train_transform == custom_transform
         assert datamodule.eval_transform is None
-        engine._setup_transform(model, datamodule=datamodule)  # noqa: SLF001
+        Engine._setup_transform(model, datamodule=datamodule)  # noqa: SLF001
         # after calling setup, train_transform should be the custom transform and eval_transform should be the default
         assert datamodule.train_transform == custom_transform
-        assert isinstance(datamodule.eval_transform, Resize)
-        assert datamodule.eval_transform.size == default_size
+        assert datamodule.eval_transform is None
+        assert model.transform == custom_transform
+
+    # test if the user-specified transform is used when passed to the datamodule
+    def test_user_specified_eval_transform(self) -> None:
```

Comment:
Maybe? Following the variable name defined in the test?
```suggestion
    def test_custom_eval_transform(self) -> None:
```